pgm-1.txt

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+echo "Enter a pid:"
+read pid
+min=1
+max=32768
+if [ $pid -ge $min -a $pid -le $max ]
+	then
+		if ps -p $pid > /dev/null 2>&1
```

Comment:
What does > /dev/null mean? 
What does 2>&1 mean?
Why are the above 2 specifications required?